README.md

```diff
@@ -1,21 +1,205 @@
+
+### Clone and set up the repo
+
+1. To run the models you will first need to setup the project. Follow the below two steps to do this:
+
+```shell
+$ git clone https://github.com/nestauk/cci_nepal
+$ cd cci_nepal
+```
+
+2. Run the command `make install` to create the virtual environment and install dependencies
+
+3. Inside the project directory run `make inputs-pull` to access the data from S3 (for those with access to the Nesta S3 account)
+
+To note the project is setup using the Nesta Cookiecutter (guidelines on the Nesta Cookiecutter can be [found here](https://nestauk.github.io/ds-cookiecutter/structure/)).
+
+The below two options depend on if you have access to real survey data or need to generate dummy data to save and run the model.
+
+### OPTION A - With access to the real survey data
+
+**Step 1: Save the file into `inputs/data`**
+<br>
+When saving your file make sure to save it in `xlsx` format.
+
+**Step 2: Update the file name in config**
+<br>
+Navigate to `cci_nepal/config` and open the `base.yaml` file. In that file you will see the below `file` variable:
+
+```shell
+data:
+  file: "dummy_data"
+```
+
+Change the value from `dummy_data` to the name of your file.
+
+### OPTION B - Without access to the real survey data
+
+#### Create a dummy dataset
+
+Run the below python file to create and save a dummy dataset that can be used for modelling. This is based on the questions used in our survey.
+
+```shell
+$ cd cci_nepal/pipeline
+$ python3 dummy_data.py
+```
+
+##### Outputs
+
+Running the `dummy_data.py` file saves a dummy version of the data you can use for modelling. The values are assigned randomly from the list of values for each column.
+
+`dummy_data.xlsx`\* saved in `inputs/data`.
+
+\*this is the default file used when you clone the repo. If you change the config `file` variable in option A you just need to remember to change it back to `dummy_data` if you want to re-run the script using your generated dummy data.
+
+### Save and run the models
+
+Perform the following steps to train and run the models:
+
+Split the survey data into training / validation and test sets
+
+```shell
+$ cd cci_nepal/pipeline
+$ python3 data_splitting_survey.py
+```
+
+#### Outputs
+
+There are three files created from running the `data_splitting.py` file. These are saved in `outputs/data/data_for_modelling` and are listed below. These form the training, validation and test sets used for modelling.
+
+- `train.csv`
+- `val.csv`
+- `test.csv`
+
+Move into the `model_workflow` folder and run the following file to train, save and run the models.
+
+```shell
+$ cd cci_nepal/pipeline/model_workflow
+$ python3 model_save.py
+$ python3 model_test.py
+```
+
+### Final Outputs
+
+There are four files created from running the models and saved to `outputs/data/test_evaluation_results`:
+
+- `shelter_test_predictions.xlsx`
+- `wash_test_predictions.xlsx`
+- `shelter_test_evaluation.xlsx`
+- `wash_test_evaluation.xlsx`
```

Comment:
If it's an easy change then csvs are preferable to xlsx as outputs (anyone can open them, no additional packages needed, can read in a text editor etc.)